server.coffee

```diff
@@ -111,15 +119,29 @@ loadApp = (logger, loadedConfig) ->
     app.get "#{ APP_ROOT }/v1/health-check", (req, res) ->
       res.send('OK\n')
 
-    port = process.env.PORT ? loadedConfig.get('server.port').get() ? 5000
-    app.listen(port)
-
-    logger.log('Server listening on port %d in %s mode', port, app.settings.env)
+    if loadedConfig.get('server.https.options').get() instanceof Object
+      https = require 'https'
+      fs = require 'fs'
+      httpsOptions = _.mapObject loadedConfig.get('server.https.options').get(), (v, k) ->
+        if _.isString(v)
```

Comment:
I reviewed this with a co-worker and he agreed with you.

He gave one suggestion for keeping the yaml format, which would be to pass an object containing a specified key, such as filename, with the path.

Example:

```yaml
server:
  port: 5999
  appRoot: "/bucky"
  https:
    port: 5599
    options:
      key:
        filename: "ssl/key.pem"
      cert:
        filename: "ssl/cert.pem"
```